test/e2e/README.md

```diff
@@ -0,0 +1,50 @@
+# Pre-requisites
+
+The test doesn't yet manage the lifecycle of the cluster being tested,
+therefore the test hosts' kubeconfig must point to a Kubernetes cluster.
+You can verify that the kubeconfig is setup correctly simply with
+`kubectl get nodes`.
+
+The cluster under test must meet the following requirements:
+* Have 3 nodes
+* Each node must be configured per the quick start:
+  * At least 512 2MiB hugepages available
+  * Each node must be labelled for use by mayastor (ie "openebs.io/engine=mayastor")
+
+The test host must have the following installed:
+* go (>= v1.15)
+* ginkgo (tested with v1.2)
+* kubectl (tested with v1.18)
+* helm
+
+# Setting up the cluster
+
+### Create a cluster in public cloud
+
+Use terraform script in `aws-kubeadm/` from
+[terraform repo](https://github.com/mayadata-io/mayastor-terraform-playground) to
+set up a cluster in AWS suitable for running the tests. The configuration file for
+terraform could look like this (replace docker registry value by yours):
+
+```
+cluster_name    = "my-e2e-test-cluster"
```

Comment:
As a good measure I always create testing resources so they can be easily identified. This is a small nudge to hint that, what do you think?

```suggestion
cluster_name    = "<NICKNAME>-e2e-test-cluster"
```

Reply:
indeed 👍